ga4gh/server/static/main.js

```diff
@@ -66,6 +63,133 @@ $("a[href='#gene_search']").on('shown.bs.tab', function(e) {
 
 })
 
+$("a[href='#sample_analysis']").on('shown.bs.tab', function(e) {
+    activeTab = e.target;
+
+    var tableIds = ["sample_analysis1", "sample_analysis2", "sample_analysis3", "sample_analysis4", "sample_analysis5", "sample_analysis6"];
+
```

Comment:
table id names are a bit ambiguous here. Why not use the display names?

Reply:
Changes were made, can you take a look? haha